pantheon-bundle/src/test/java/com/redhat/pantheon/asciidoctor/AsciidoctorServiceTest.java

```diff
@@ -75,16 +76,16 @@ public void testGetModuleHtmlWhenCached() throws Exception {
 
         // Given
         slingContext.build()
-                .resource("/module/locales/en_US/released/metadata")
-                .resource("/module/locales/en_US/released/content/asciidoc/jcr:content",
+                .resource("/module/en_US/released/metadata")
+                .resource("/module/en_US/released/content/asciidoc/jcr:content",
```

Comment:
:+1: I guess this was never changed, but I wonder why it's presenting problems only now :thinking:

Reply:
Line 60 failed on NoSuchElementException. Line 79-84 didn't fail. I changed it so that they are consistent.